homeassistant/components/evohome/__init__.py

```diff
@@ -119,13 +120,35 @@
 
 
 async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
-    """Create a (EMEA/EU-based) Honeywell TCC system."""
+    """Set up the Evohome integration from the configuration.yaml file."""
 
-    async def load_auth_tokens(store: Store) -> tuple[dict, dict | None]:
+    scan_interval: timedelta = config[DOMAIN].pop(CONF_SCAN_INTERVAL)
+    config[DOMAIN][CONF_SCAN_INTERVAL] = int(scan_interval.total_seconds())
+
+    # until config flow is implemented, this most closely matches the current behaviour
+    if entries := hass.config_entries.async_entries(DOMAIN):
+        hass.config_entries.async_update_entry(entries[0], data=config[DOMAIN])
```

Comment:
I think you should look for a SOURCE_IMPORT entry.

Reply:
But not needed until after I add config flow?